src/styles/starrating.css

```diff
@@ -0,0 +1,56 @@
+@font-face {
+  font-family: "Glyphicons Halflings";
+  src: url("//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/fonts/glyphicons-halflings-regular.eot");
+  src: url("//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/fonts/glyphicons-halflings-regular.eot?#iefix")
+      format("embedded-opentype"),
+    url("//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/fonts/glyphicons-halflings-regular.woff2")
+      format("woff2"),
+    url("//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/fonts/glyphicons-halflings-regular.woff")
+      format("woff"),
+    url("//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/fonts/glyphicons-halflings-regular.ttf")
+      format("truetype"),
+    url("//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular")
+      format("svg");

```

Comment:
We shouldn't use CDN by default. Can we use a Unicode char or a local font?

Reply:
A couple ideas.
- Could use Arial font which also has dingbats, but then it won't work on mobile devices. :(
- Make a custom "JSON-Edit font", using Fontello <http://fontello.com/> and then include that. This could also be use as fallback font for the button icons. (Font packages generated with Fontello can easily be modified if original archive is preserved.)
- Create separate stylesheets, matching the "iconlibs". As the glyphicons are part of Bootstrap3 (Which I used for testing). and are the same used by the iconlibs file.